api/tests/integration/infrastructure/repositories/higher-schooling-registration-repository_test.js

```diff
@@ -314,38 +257,43 @@ describe('Integration | Infrastructure | Repository | higher-schooling-registrat
           databaseBuilder.factory.buildSchoolingRegistration({ organizationId: organization.id, studentNumber: '4' });
           await databaseBuilder.commit();
 
-          const error = await catchErr(higherSchoolingRegistrationRepository.batchCreate)([higherSchoolingRegistration]);
+          await higherSchoolingRegistrationRepository.upsertStudents([{ ...higherSchoolingRegistration, lastName: 'Ishii updated' }]);
 
-          expect(error).to.be.an.instanceOf(SchoolingRegistrationsCouldNotBeSavedError);
+          const results = await knex('schooling-registrations')
+            .select('*', 'status AS studyScheme')
+            .where({ organizationId: organization.id })
+            .orderBy('studentNumber');
+
+          expect(results.length).to.equal(1);
+          expect(results[0].lastName).to.equal('Ishii updated');
         });
       });
-    });
-  });
 
-  describe('#findStudentNumbersByOrganization', () => {
-    let organization;
-    const studentNumber = '123A';
+      context('when there is schooling registrations for an oher organizationId and student number', () => {
```

Comment:
'when there is schooling registrations for an other organizationId and student number'